devkit/samples/attachments/attachments.feature

```diff
@@ -15,30 +15,26 @@ Feature: Attachments
 
     When the string "hello" is attached as "application/octet-stream"
 
+  Scenario: Log text
+    When the string "hello" is logged
+
+  Scenario: Log ANSI coloured text
+    When text with ANSI escapes is logged
+
   Scenario: Log JSON
      When the following string is attached as "application/json":
        ```
        {"message": "The <b>big</b> question", "foo": "bar"}
        ```
 
-  Scenario: Log text
-    When the string "hello" is logged
-
-  Scenario: Log ANSI coloured text
-     When text with ANSI escapes is logged
-
   Scenario: Byte arrays are base64-encoded regardless of media type
     When an array with 10 bytes is attached as "text/plain"
 
-  Scenario: Streams are always base64-encoded
```

Comment:
Have we removed the streams one because it's not applicable to Ruby?

Reply:
Because it's not doing a stream. The stream step is never used.

When it was originally added it was never then consumed. We should remove it and then re-apply it as/when we do it